eventdata/challenges/default.json

```diff
@@ -1,5 +1,6 @@
     {
       "name": "append-no-conflicts",
+      "user-info": "This track is deprecated. Please use https://github.com/elastic/rally-eventdata-track.",
```

Comment:
Can we please add this property for all challenges? Also, I suggest to change the wording slightly: `This track is deprecated and will be removed soon. ...`. Wdyt?

Reply:
I will update the wording. For other challenges should I just add this property with empty content?